ui/component/viewers/videoViewer/internal/videojs.jsx

```diff
@@ -376,6 +364,27 @@ export default React.memo<Props>(function VideoJs(props: Props) {
         vjsPlayer = window.player;
       }
 
+      // hide unused elements on livestream
+      if (isLivestream) {
+        vjsPlayer.addClass('vjs-live');
+        vjsPlayer.addClass('vjs-liveui');
+        // $FlowIssue
+        vjsPlayer.controlBar.currentTimeDisplay?.el().style.setProperty('display', 'none', 'important');
+        // $FlowIssue
+        vjsPlayer.controlBar.timeDivider?.el().style.setProperty('display', 'none', 'important');
```

Comment:
<img width="134" alt="image" src="https://user-images.githubusercontent.com/64950861/175932748-7b7c8c68-5f85-4f88-8a47-970a06cd7983.png">

Misaligned, no red dot, not clickable to reset to live